.env.example

```
@@ -106,6 +106,44 @@ BREVO_LIST_ID=
+TWILIO_AUTH_TOKEN=your_twilio_auth_token
+# Verify service SID: Twilio Console → Verify → Services → create or use default
+TWILIO_VERIFY_SERVICE_SID=VAxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
+
+# =============================================================================
+# SmileID KYC Verification Services
+# =============================================================================
+
+SMILE_IDENTITY_BASE_URL=XXXXXX #get from docs
+SMILE_IDENTITY_API_KEY=your_api_key_here
+SMILE_IDENTITY_PARTNER_ID=your_partner_id_here
+SMILE_ID_CALLBACK_URL=  #optional callback url
```

dotenv-linter (4.0.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)